packages/pwa-buildpack/lib/cli/create-project.js

```diff
@@ -222,10 +222,11 @@ module.exports.handler = async function buildpackCli(argv) {
     if (process.env.DEBUG_PROJECT_CREATION) {
         prettyLogger.info('Debug: Removing generated tarballs');
         const pkgDir = require('pkg-dir');
-        const monorepoDir = resolve(pkgDir.sync(__dirname), '../../');
+        const packagesDirectory = resolve(pkgDir.sync(__dirname), '../');
         prettyLogger.info(
-            execa.shellSync('rm -v packages/*/*.tgz', { cwd: monorepoDir })
-                .stdout
+            execa.shellSync('find . -name "*.tgz" -delete -print', {
+                cwd: packagesDirectory
+            }).stdout
         );
     }
```

Comment:
I don't know why I didn't think of this before, but...what if we just put tarballs in package roots in `.gitignore`? I just tested it and it worked fine. We could just remove this section in `create-project.js` entirely. Which would be nice, since now we're not looking for DEBUG_PROJECT_CREATION in two places.

Gitignore additions:
```gitignore
## May temporarily create tarballs during scaffolding debug
packages/*/*.tgz
packages/extensions/*/*.tgz
```

Reply:
Got rid of cleanup and added a gitignore rule in 9a0ce84